src/amchart/CommonSerial.js

```diff
@@ -260,27 +285,27 @@
         this._chart.dataProvider = this.amFormatData(this.data());
         this.amFormatColumns();
 
-        this._chart.valueAxes[0].title = this.yAxisTitle();
-        this._chart.valueAxes[0].titleColor = this.yAxisTitleFontColor();
-        this._chart.valueAxes[0].titleFontSize = this.yAxisTitleFontSize();
-        this._chart.valueAxes[0].axisThickness = this.axisLineWidth();
-        this._chart.valueAxes[0].color = this.yAxisFontColor();
-        this._chart.valueAxes[0].fontSize = this.axisFontSize();
-        this._chart.valueAxes[0].axisColor = this.yAxisBaselineColor();
-        this._chart.valueAxes[0].axisAlpha = this.axisAlpha();
-        this._chart.valueAxes[0].fillColor = this.yAxisFillColor();
-        this._chart.valueAxes[0].fillAlpha = this.yAxisFillAlpha();
-
-        this._chart.valueAxes[0].gridAlpha = this.yAxisGridAlpha();
-        this._chart.valueAxes[0].dashLength = this.yAxisDashLength();
-        this._chart.valueAxes[0].boldPeriodBeginning = this.yAxisBoldPeriodBeginning();
-        this._chart.valueAxes[0].axisTitleOffset = this.yAxisTitleOffset();
+        this._chart.valueAxes[0].title = this.yAxis()[0].yAxisTitle();
```

Comment:
eventually loop through the axis to set the values .... axis can be delete like so : this.xAxis().slice(1,1);